commands/apps_dev.go

```diff
@@ -83,39 +92,40 @@ func AppsDev() *Command {
 	AddStringFlag(
 		build, doctl.ArgEnvFile,
 		"", "",
-		"Additional environment variables to inject into the build.",
+		"An optional path to a .env file with overrides for values of app spec environment variables.",
 	)
 
 	AddBoolFlag(
 		build, doctl.ArgNoCache,
 		"", false,
-		"Whether or not to omit the cache for the build.",
+		"Set to disable build caching.",
 	)
 
 	AddStringFlag(
 		build, doctl.ArgBuildCommand,
 		"", "",
-		"Optional build command override for local development.",
+		"An optional build command override for local development.",
 	)
 
 	AddDurationFlag(
 		build, doctl.ArgTimeout,
 		"", 0,
-		"An optional timeout duration for the build",
+		`An optional timeout duration for the build. Valid time units are "ns", "us" (or "µs"), "ms", "s", "m", "h". Example: 15m30s`,
```

Comment:
You could probably drop the small time values, they seem impractical, to make this more concise and reduce clutter.  
```suggestion
		`An optional timeout duration for the build. Valid time units are "s", "m", "h". Example: 15m30s`,
```